cmd/argo/commands/list.go

```diff
@@ -64,6 +64,34 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows",
+		Example: `# List all workflows:
+  argo list
+
+# Show workflows from all namespaces:
+  argo list -A
+
+# List all completed worflows:
+  argo list --completed
+
+# List completed workflows finished before 2h:
+  argo list --older 2h
+
+# List worflows with more information (such as parameters):
+  argo list -o wide
+
+# List worflows in yaml format:
+  argo list -o yaml
+
+# List all running worflows:
+  argo list --running
+
+# List worflows with specific labels:
```

Comment:
```suggestion
# List workflows that have both labels:
```

- we should be specific to the example command. "both" also specifies that [label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors) are ANDed together
- typo: missing "k"